tools/CMakeLists.txt

```diff
@@ -268,11 +283,20 @@ function(pico_load_map_clear_sram TARGET)
     )
 endfunction()
 
-# pico_set_binary_version(<TARGET> [MAJOR <version>] [MINOR <version>] [ROLLBACK <version>] [ROLLBACK_ROWS <rows...>])
+# pico_set_binary_version(<TARGET> [MAJOR <version>] [MINOR <version>] [ROLLBACK <version>] [ROWS <rows...>])
```

Comment:
I've just noticed that this one says `pico_set_binary_version(<TARGET> ...)` wheres all the other CMake signatures say e.g. `pico_set_otp_key_output_file(TARGET ...)` ?